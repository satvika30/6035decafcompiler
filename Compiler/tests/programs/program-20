class Program {
	int seed, m, c, a;
	
	int random() {
		seed = (a*seed+c)%m;
		return seed;
	}
	
	void main() {
		int i;
		seed = 17;
		m = 47;
		c = 123;
		a = 6;
		for i = 0, 10 {
			callout("printf","%d\n",random());
		}
	}
}