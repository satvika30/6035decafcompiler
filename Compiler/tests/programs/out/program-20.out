Randomizations:

One output:
37
16
31
27
3
0
29
15
25
38